dataactcore/scripts/loadDUNS.py

```diff
@@ -117,7 +117,7 @@ def get_parser():
                         process_from_dir(root_dir, sorted_monthly_file_names[0],
                                          sess, local, sftp, monthly=True, benchmarks=benchmarks)
 
-                        get_duns_batches(wdsl_client, sess, updated_date=updated_date)
+                        # get_duns_batches(wdsl_client, sess, updated_date=updated_date)
```

Comment:
Are these commented out for a reason? If so, should we just remove them entirely? If not, should we uncomment them?

Reply:
They were originally there to pull in parent recipient duns data (via the SAM service) but since we can pull that data directly from the CSVs we dont need them anymore. I can remove them.